### PR TITLE
Add strip() example

### PR DESCRIPTION
**Pull request by FedericoCeratto**

### CODE REVIEW

**Comment by Varriount** [comment]
@FedericoCeratto Any idea why one of the checks failed?

**Comment by FedericoCeratto** [comment]
@Varriount I'm seeing various errors related to async and a compiler crash. They don't seem to be related to the change.

**Comment by dom96** [comment]
It seems clang segfaulted: https://travis-ci.org/nim-lang/Nim/jobs/390168729#L7887

Why are we using clang on Linux?

File: lib/pure/strutils.nim [hunk 1]
```diff
@@ -326,6 +326,9 @@ proc strip*(s: string, leading = true, trailing = true,
   ## If `leading` is true, leading `chars` are stripped.
   ## If `trailing` is true, trailing `chars` are stripped.
   ## If both are false, the string is returned unchanged.
+  ## .. code-block:: nim
```
Comment (dom96) [inline lib/pure/strutils.nim]: This needs an empty space above it. It won't be shown correctly otherwise.

**Comment by dom96** [comment]
Here is the PR that introduces ``clang`` in travis: https://github.com/nim-lang/Nim/pull/3403

FWIW this has been changed by @Araq in the meantime, so just **rebase please**.

File: lib/pure/strutils.nim [hunk 2]
```diff
@@ -330,6 +330,8 @@ proc strip*(s: string, leading = true, trailing = true,
   ## If `leading` is true, leading `chars` are stripped.
   ## If `trailing` is true, trailing `chars` are stripped.
   ## If both are false, the string is returned unchanged.
+  runnableExamples:
+    echo " vhello ".strip().strip(trailing=false, chars={'v'})
```
Comment (Araq) [inline lib/pure/strutils.nim]:
```nim
runnableExamples:
  doAssert " vhellov ".strip().strip(trailing = false, chars = {'v'}) == "hellov"
```

**Pull request merged**